v1/jenkins/inteledison_c.sh

```diff
@@ -2,7 +2,7 @@
 # Copyright (c) Microsoft. All rights reserved.
 # Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-build_root=$(cd "$(dirname "$0")/.." && pwd)
+build_root=$(cd "$(dirname "$0")/../v1" && pwd)
```

Comment:
Does this work? `"$0"` will return the full path to the bash script (`v1/jenkins/inteledison_c.sh`), `dirname` will filter out the filename (so, `v1/jenkins/`), the `cd` command will navigate up to the parent (so, `v1/`) and then down to a `v1/` subdirectory (so, `v1/v1/`).

Since that directory doesn't exist, I'd expect this command to fail.

Reply:
Good point, I put this in when I moved the Jenkins directory...  I need to update this, and double-check the others.